app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java

```diff
@@ -160,7 +161,12 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) throw
             }
 
             if (mPDFUtils.reorderRemovePDF(mPath, outputPath, pages)) {
-                viewPdfButton(outputPath);
+                if (sameFile) {
```

Comment:
There is `ArrayList<Bitmap> bitmaps = new ArrayList<>();` in this file.
Can't this be used for comparison rather than this approach?